src/lib/redux/userData.ts

```diff
@@ -0,0 +1,10 @@
+import { store } from './store';
+import { saveUserData, setLoadingForUserData } from './userDataSlice';
+
+// TODO: Change from ANY type to user schema after Maria's schema PR gets merged
```

Comment:
Maria's PR has since been merged if you want to update this!

Reply:
Looks like the typing isn't as easy as I thought since the fields surprisingly don't match :( . I updated the TODO and will do it another time!